examples/pet-mad-nc/data/lammps-respa.in

```diff
@@ -0,0 +1,30 @@
+units metal
+
+atom_style atomic
+
+read_data bmimcl.data
+
+mass 1 12.011
+mass 2 35.453
+mass 3 1.00784
+mass 4 14.0067
+
+pair_style hybrid/overlay metatomic_1 pet-mad-latest.pt device cpu non_conservative on scale 1.0 metatomic_2 pet-mad-latest.pt device cpu non_conservative on scale -1.0 metatomic_3 pet-mad-latest.pt device cpu non_conservative off scale 1.0
```

Comment:
if using the non-kokkos version, should this use `hybrid/scaled` instead? I'd like to be able to remove the `scale` option once kokkos is fixed

Reply:
@ceriottm is there a specific reason why you merged with `hybrid/overlay` instead of `hybrid/scaled`? If not, I'll send a PR to update it